### ISSUE TITLE
Who wants to write a Zmodem or Ymodem protocol?

### ISSUE THREAD

**Issue by ThaDoctor72** [Pinacolada64/ImageBBS#51]
Probably more of a feature request, but I don't have access to write into that area.  I'm pretty sure the C-Net 128 software has Ymodem, and I wonder if it can be ported over for our purposes.  Since so many people are running emulators to call and download from us, it seems almost a shame to limit them to Xmodem only.

**Comment by Pinacolada64** [comment]
The `wishlist` tag is what I came up with. And you're right, X/Ymodem would be nice. I don't know if it would fit in 2,680 bytes (the maximum allowable `++ protocol` size right now). We could look at C-Net 128, or even Nick Rossi's [_NovaTerm 10_](http://www.kludgesoft.com/c64/nt10.html) code. Not sure where source is for C-Net 128. Any leads?

I do want to fix #46 first. (I mean, actually fixing 19.2k, not the careless commits I make [_real close to that, though_]).

**Comment by ThaDoctor72** [comment]
Agreed. A stable connection and DTR availability trumps UD protocol any day.
